### DISCUSSION TITLE
Use Bitbucket private repository in codemagic

### DISCUSSION THREAD

**Discussion by chaitox** [codemagic-ci-cd/codemagic-docs#1831] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Add in codemagic.yaml 
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/20906071/8faccba0-984c-4881-8226-15322f417d11)
2. Group added here
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/20906071/7814ed95-245a-4d39-b3a7-91ca9bffcb99)
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/20906071/bfb0c17c-75e3-4d7f-85e1-22b9a3c12364)

### Expected results

Continie de build generating de APK.

### Actual results

![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/20906071/3b2a5f42-e43d-4e55-ba05-f9d928b2fc03)

**Comment by dtrdic** [comment]
Hey @chaitox can you please elaborate a little bit more on what you want to achieve?
if you want to add a private repository to the Codemagic platform you can do it in a different way, no need to create this MODULE_SSH_KEY variable.
Just click on Add application, connect the repository using the "**Other**" option, and enter all information that is needed
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/38918425/f6b79c62-8f0c-499b-a7ca-e0dc1c97fbb3)

Here is a guide for reference to check how to connect your repository via SSH using the 'Other' option
https://docs.codemagic.io/getting-started/adding-apps/

Please let me know if I misunderstood you, or if something is not clear in the docs

**Reply by chaitox** [reply]
Hello @dtrdic, in my flutter proyect I have a private repository in the pubspec.yaml like this:
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/20906071/25533930-2e85-468f-8b32-fa2e8af412ce)
but when I try to build in codemagic got this error:
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/20906071/0581ba1d-0bb5-4dfe-b624-c19646965e2e)
